CONTRIBUTING.md

```diff
@@ -4,15 +4,46 @@ This document provides information about contributing to the [Keptn documentatio
 This documentation is authored with [markdown](https://www.markdownguide.org/basic-syntax/)
 and rendered using the [Hugo static website generator framework](http://gohugo.io).
 
+We welcome and encourage contributions of all levels.
+You can make modifications using the github editor;
+this works well for small modifications but, if you are making significant changes, you may find it better to fork and clone the repository and make changes using the text editor or IDE of your choice
+You can run the Hugo formatter locally to check the rendered documentation
```

Comment:
```suggestion
You can also run the Hugo based website locally to check the rendered documentation
```

Reply:
You cannot make modifications using GitHub.com as the commits aren't signed and the current process complains if not all commits are signed.